src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/DropColumns.cs

```diff
@@ -20,11 +22,14 @@ internal sealed class DropColumnsFunction : FunctionWithTableInput
     {
         public override bool IsSelfContained => true;
 
+        public override bool HasLambdas => true;
+
         public override bool SupportsParamCoercion => false;
 
         public DropColumnsFunction()
             : base("DropColumns", TexlStrings.AboutDropColumns, FunctionCategories.Table, DType.EmptyTable, 0, 2, int.MaxValue, DType.EmptyTable)
         {
+            ScopeInfo = new FunctionScopeInfo(this);
```

Comment:
>ScopeInfo [](http://example.com/codeflow?start=12&length=9)

?

Reply:
Needed for functions having lambdas or identifiers